src/projectviewerwindow.cpp

```diff
@@ -0,0 +1,67 @@
+#include "projectviewerwindow.h"
```

Comment:
Please, move projectviewerwindow.* files to separate folder, add a .pri file that include all that files to project.

Reply:
Done. Please, check the project how(maybe I made mistakes again)